Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -309,7 +309,10 @@ private extension HTTPClient {
         if let response = response {
             switch response {
             case let .success(response):
-                Logger.debug(Strings.network.api_request_completed(request.httpRequest, httpCode: response.statusCode))
+                Logger.debug(Strings.network.api_request_completed(
+                    request.httpRequest,
+                    httpCode: urlResponse?.httpStatusCode ?? response.statusCode
```

Comment:
let's add a comment, it's not obvious at a glance why we're checking them in this order

Reply:
or make it a method with a clear name or something

Reply:
Yup good point 👍🏻